test/unleash.network.test.js

```diff
@@ -31,5 +31,5 @@ test.cb('should emit network errors', t => {
         process.nextTick(() => {
             t.end();
         });
-    }, 0);
+    }, 5);
```

Comment:
Had to bump this, or `destroy` was called before the `fetch` to `client/features` was completed, which meant the error handler was removed, which means an unhandled error bubbled up to node, which causes ava to fail the test (good change!)